src/main/java/catan/services/impl/PlayServiceImpl.java

```diff
@@ -138,6 +149,17 @@ private void throwDice(GameBean game) {
         game.setDiceThrown(true);
     }
 
+    private void buyCard(GameUserBean gameUser, GameBean game, Map<String, String> returnedParams) throws PlayException, GameException {
+        DevelopmentCards availableDevelopmentCards = game.getAvailableDevelopmentCards();
+        DevelopmentCard obtainedDevelopmentCard = cardUtil.pullDevelopmentCard(availableDevelopmentCards);
```

Comment:
obtainedDevelopmentCard  - it is not obtained yet, it will be obtained after you give this resource to player